public/docs/ts/latest/guide/index.jade

```diff
@@ -1,10 +1,501 @@
++makeExample('intro/ts/app/hero-list.component.ts', 'ctor')
+:marked
+  When Angular creates a component, it first asks an **Injector** for
+  the services that the component requires. 
+  
+  An `Injector` maintains a container of service instances that it has previously created.
+  If a requested service is not in the container, the injector makes one and adds it to the container
+  before returning the service to Angular. 
+  When all requested servics have been resolved and returned, 
+  Angular can call the component's constructor with those services as arguments.
+  This is what we mean by *dependency injection*.
+  
+  The process of `HeroService` injection looks a bit like this:
+figure
+  img(src="/resources/images/devguide/intro/injector-injects.png" alt="Service" )
+:marked
+  If the `Injector` doesn't have a `HeroService`, how does it know how to make one?
+  
+  In brief, we must have previously registered a **provider** of the `HeroService` with the `Injector`.
+  A provider is something that can create or return a service, typically  the service class itself.
+  
+  We can register providers at any level of the application component tree.
+  We often do so at the root when we bootstrap the application so that
+  the same instance of a service is available everywhere.
++makeExample('intro/ts/app/boot.ts', 'bootstrap')
+:marked
+  Alternatively, we might register at a component level ...
++makeExample('intro/ts/app/hero-list.component.ts', 'providers')
+:marked
+  ... in which case we get a new instance of the
+  service with each new instance of that component.
+  
+  We've vastly over-simplified dependency injection for this overview.
+  We can learn the full story in the [Dependency Injection](dependency-injection.html) chapter.
+  
+  The points to remember are:
+  * dependency injection is wired into the framework and used everywhere.<br><br>
+  * the `Injector` is the main mechanism.
+    * an injector maintains a *container* of services that it created.
+    * an injector can create a new service instance from a *provider*.
+  * a *provider* is a recipe for creating a service.
+ 
+  * we register *providers* with injectors.
+
+<a id="other-stuff"></a>  
+.l-main-section
+:marked
+  # The Other Stuff
+  
+  We've learned just a bit about the seven main building blocks of an Angular application
+  1. [Component](#component)
+  1. [Template](#template)
+  1. [Metadata](#metadata)
+  1. [Data Binding](#data-binding)
+  1. [Directive](#directive)
+  1. [Service](#service)
+  1. [Dependency Injection](#dependency-injection)
+  
+  That's a foundation for everything else in an Angular application
+  and it's more than enough to get going.
+  But it doesn't include everything we'll need or want to know.
+   
+  Below is a brief, alphabetical list of other important Angular features and services. 
+  Most of them are covered in this Developers Guide (or soon will be):
```

Comment:
Reminds me that the current title is "Developers Guides" which is totally confusing!